pom.xml

```diff
@@ -769,6 +769,7 @@
         <configuration>
           <configLocation>dev-support/checkstyle.xml</configLocation>
           <failOnViolation>false</failOnViolation>
+          <includeTestSourceDirectory>true</includeTestSourceDirectory>
```

Comment:
@leo65535 , thanks for working on this.  Fixing the checkstyle warnings in test is good.  However, let's don't run checkstyle for the tests.  We should focus more on the main code instead of spending too much energy on the tests.

Reply:
Make sense. But it's better to keep the same rules for both `source` and `test` code.